File: apps/institution-api/Models/Dto/SpecializationCostDto.cs
using System;

namespace ProviderApi.Models.Dto
{
  public class SpecializationCostDto
  {
    public int SpecializationId { get; set; }
    public int? ProgramId { get; set; }
    public int? ProviderId { get; set; }
    public int SessionOfStudyId { get; set; }
    public string FundingYearCode { get; set; }
    public DateTime? SessionStartDate { get; set; }
    public DateTime? SessionEndDate { get; set; }
    public string CostType { get; set; }
    public decimal? Amount { get; set; }
    public decimal? StudyYear { get; set; }
    public bool? DisplayOnSfsflag { get; set; }
  }
}

File: libs/common/models/specialization-cost-dto.ts



export class SpecializationCostDto {
    public specializationId: number = 0;
    public programId: number;
    public providerId: number;
    public sessionOfStudyId: number = 0;
    public fundingYearCode: string = '';
    public sessionStartDate: Date;
    public sessionEndDate: Date;
    public costType: string = '';
    public amount: number;
    public studyYear: number;
    public displayOnSfsflag: boolean;

    constructor(init? : Partial<SpecializationCostDto>) {
        
        Object.assign(this, init);
    }
}

